index.js

```diff
@@ -238,6 +45,8 @@ module.exports = function (config, options) {
   });
 
   // Clear cache on re-runs
+  // TODO: update deprecated event name v0.11.0
+  // https://www.11ty.dev/docs/events/#eleventy.beforewatch
```

Comment:
Do we need to change this to watch the new `eleventy.beforeWatch` as well as the old `beforeWatch`?

Reply:
Might be worth it, although its deprecated rather than removed. I suspect it will go in v3. I've left it for now - I thought I might look at options for detecting the 11ty version the plugin is running against.